experiments/vision-transformer/unetr/livecell/common.py

```diff
@@ -40,14 +53,100 @@ def _get_output_channels(with_affinities):
     return n_out
 
 
+def get_distance_maps(labels):
+    labels = vigra.analysis.relabelConsecutive(labels.astype("uint32"))[0]
+    # compute the eccentricty centers
+    # First expensive step: center computation (leave it here, it's done once per image)
+    centers = vigra.filters.eccentricityCenters(labels.astype("uint32"))
```

Comment:
This also computes the eccentricity center for the background.

Reply:
Ah interesting. That's good to know, I will note it for now. Thanks!